### ISSUE TITLE
[css-anchor-position] Naming of the `inside` and `outside` `<anchor-side>` values

### ISSUE THREAD

**Issue by jwatt** [w3c/csswg-drafts#11052]
When I first read the description of the `inside` and `outside` values for [`<anchor-side>`](https://drafts.csswg.org/css-anchor-position-1/#typedef-anchor-side) I initially thought they required some magical implicit behavior to take the final position and size of the anchored element, then shift it to be inside/outside the anchor box...somehow. I think that's because the of the common English definitions of the words `inside` and `outside`.

I suspect that at least some content authors will also initially be confused by the names of these values. Would `own` and `opposite` (or `own-side` and `opposite-side`) perhaps be less prone to confusion?

**Comment by jwatt** [comment]
Or probably cleaner would be `near` (edge) and `far` (edge) IMO.

**Comment by tabatkins** [comment]
The names were already changed from `auto`/`opposite`; the intention is that they attach you to the relevant edge that puts you "inside" or "outside" the anchor. In particular, `inset: anchor(inside);` works reasonably well, but one-off inset edges with `outside` work well too afaict: `left: anchor(outside)` is "set my left edge to be on the outside of the anchor", aka on the anchor's right side.

**Comment by jwatt** [comment]
Yeah, I get it. They're named with a resulting **effect** in mind given suitable values for **other** properties. But these keywords set a single edge to a length that matches a particular position, the position being another edge. That edge itself is not "inside" or "outside" anything. The other keywords for `<anchor-side>` are clearly positions, but these keywords are not, which I still think will end up confusing some other people too.

To me, something like `inset: anchor(near-edge);` seems as good as `inset: anchor(inside);`, and `left: anchor(far-edge)` seems as good as `left: anchor(outside)`. (shrug)

Regarding your current examples working well, that relies on authors not thinking there's an inference that the names involve side effects to get a particular effect, ending up trying things like `left: anchor(outside); right: anchor(right);` (result in zero width).
